website/content/docs/security/acl/acl-multiple-datacenters.mdx

```diff
@@ -0,0 +1,234 @@
+---
+layout: docs
+page_title: ACL in Multiple Datacenters
+description: >-
+  When Consul ACLs are enabled in multiple datacenters that are federated together,
+  specific bootstrapping of ACLs and policies are needed
+---
+
+# ACLs in Federated Datacenters
+
+This topic describes how to set up Consul's access control list (ACL) system
+in cluster deployments that span multiple data centers. This documentation is applicable
+to new clusters rather than existing clusters.
+
+-> **1.4.0 and later:** This guide only applies in Consul versions 1.4.0 and later.
+
+## Configure ACLs in the Primary Datacenter
+
+In a federated Consul deployment, one of the datacenters is marked as the primary datacenter.
+The `acl` configuration block should be added to the primary datacenter server's configuration file
+as shown in the following example.
+
+See the [ACL Config Stanza](https://www.consul.io/docs/agent/options#acl) for more detailed descriptions of each option.
+
+-> **Versions before 1.11.0:** The `initial_management` token was called the `master` token in versions
+prior to 1.11.0
+
+<CodeTabs heading="ACL Configuration in Primary">
+
+```hcl
+bootstrap_expect =  3
+primary_datacenter =  "PRIMARY_DATACENTER_VALUE"
+acl = {
+ enabled =  true
+ default_policy = "deny"
+ down_policy = "deny"
+ enable_token_persistence = true
+ enable_token_replication = true
+ tokens = {
+   initial_management =  "ACL_MANAGEMENT_TOKEN"
+   agent = "YOUR_ACL_AGENT_TOKEN"
+  }
+}
+
+```
+
+```json
+{
+  "bootstrap_expect": N,
+  "primary_datacenter": "PRIMARY_DATACENTER_VALUE",
+  "acl": {
+    "enabled": true,
+    "default_policy": "deny",
+    "down_policy": "deny",
+    "enable_token_persistence": true,
+    "enable_token_replication": true,
+    "tokens": {
+      "initial_management": "ACL_MANAGEMENT_TOKEN",
+      "agent": "ACL_AGENT_TOKEN"
+    }
+  }
+}
+```
+
+</CodeTabs>
+
+~> **Warning:** Note that most enterprise deployments have security requirements that prevent specifying tokens in configuration files.
+Additionally, in the above config `enable_token_persistence` flag is set, so the token is stored to disk in the agent's
```

Comment:
```suggestion
The `enable_token_persistence` flag is also set in the configuration example so that the token is stored to disk in the agent's
```
Just trying some different wording to see if we can improve clarity; not married to it